docs/features/message-pumps/service-bus.md

```diff
@@ -106,8 +106,8 @@ public void ConfigureServices(IServiceCollection services)
             options.JobId = Guid.NewGuid().ToString();
 
             // Indicate whether or not a new Azure Service Bus Topic subscription should be created/deleted
-            // when the message pump starts/stops (default: CreateOnStart).
-            options.TopicSubscription = TopicSubscription.CreateOnStart | TopicSubscription.DeleteOnStop;
+            // when the message pump starts/stops (default: DeleteOnStop, which implicitly means also 'create on start').
+            options.TopicSubscription = TopicSubscription.DeleteOnStop;
```

Comment:
Ha think there is some miscommunication, I'd keep both by default.
```suggestion
            options.TopicSubscription = TopicSubscription.CreateOnStart | TopicSubscription.DeleteOnStop;
```

Reply:
Yes, I know 😄 . `CreateOnStart = 1` and `DeleteOnStop = 3` so bitwise, if you choose `DeleteOnStop` you implicitly say that you want to create it on start. So it's the same.

Reply:
But in the docs, we could make explicit indeed. In code, not very much added value I think.

Reply:
Frankly, it should not be that way, how would I then delete it on stop without creating it here if I want?

Reply:
It's implicit saying that you will create it at start. It's actually kind of a failsafe. So, that you can't configure the pump with only deleting the subscription, without also implicitly stating that you want it to be created at startup.
For readability, I can understand that it makes sense to use them both, but behind the scenes, there's no difference.

Reply:
I don't like hidding things, we should just use a flag approach and if only `DeleteOnStop` is specified only do that. If the subscription is not there, throw an exception, it's their fault since they've configured it that way 🤷‍♂️

But default should be `CreateOnStart | DeleteOnStop` indeed.

Reply:
Reason is - Sometimes subscriptions are created externally so we should not implicitly do this if it's not specified.